lib/Logger/Logger.h

```diff
@@ -31,7 +29,7 @@ class Logger {
          */
         static void init(const uint16_t port);
 
-        Logger(Logger const&) = delete;
+        Logger(Logger const&)         = delete;
```

Comment:
toooo much whitespace

Reply:
See above, `=` alignment currently active.

Reply:
Do we really need this alignment? I think it's pretty unusual and not really helpful

Reply:
Indeed a bit unusual - but we also have unusually long lists of preprocessors directives, global variables and all. I found it helpful in reading - but can of course drop it if you wish.

Reply:
Then I would take it out!

Reply:
Done.